packages/kit-bg/src/services/ServicePrimeTransfer/ServicePrimeTransfer.ts

```
@@ -1,5 +1,6 @@
+/* eslint-disable spellcheck/spell-checker */
 import { Semaphore } from 'async-mutex';
-import { debounce, isEmpty, isNaN, isNil, uniqBy } from 'lodash';
+import { cloneDeep, debounce, isEmpty, isNaN, isNil, uniqBy } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)